packages/tasit-action/src/DecentralandEstate.test.js

```diff
@@ -0,0 +1,12 @@
+import DecentralandEstate from "./DecentralandEstate";
+
+import { local as localAddresses } from "../../tasit-contracts/decentraland/addresses";
+const { EstateRegistry: ESTATE_ADDRESS } = localAddresses;
+
+describe("TasitAction.DecentralandEstate", () => {
+  it("should get the Estate owner", async function() {
+    const estate = new DecentralandEstate(ESTATE_ADDRESS);
+    let owner = await estate.owner();
+    expect(owner).to.equal("0x0000000000000000000000000000000000000000");
```

Comment:
Why is this what ends up being the owner? Because it's unowned? Or is that the default with ganache?

Reply:
And how does it relate to the land contract having an owner?

Reply:
It's unowned at this point because this test runs against an uninitialized contract.

Reply:
Ah, right. Okay. Should we make a TODO to change that, or does the test serve its purpose fine with it uninitialized?

Reply:
I think that serves for now, but I've added a comment to improve that when we move to a forked from testnet blockchain.

Reply:
Okay, sounds good. Note that I still think we'll keep our original unit tests the same even once we've done that